master/buildbot/process/buildstep.py

```diff
@@ -496,9 +496,6 @@ def setRenderable(res, attr):
         except BuildStepFailed:
             self.results = FAILURE
 
-        except BuildSlaveTooOldError:
```

Comment:
I spent a faire amount of time debugging why my step went in exception with no message. the generic except Exception catches and reports the error much more nicely